### PR TITLE
Fixed typo in the function published return type

### PR DESCRIPTION
**Pull request by teclator**
It was introduced in #27

### CODE REVIEW

File: src/modules/HttpServerWidgets.rb [hunk 1]
```diff
@@ -3651,7 +3651,7 @@ def initSummaryText(key)
     publish :function => :initVhostDetails, :type => "void (string)"
     publish :function => :handleVhostDetails, :type => "symbol (string, map)"
     publish :function => :storeVhostDetails, :type => "void (string, map)"
-    publish :function => :widgets, :type => "map <string, map <string, any>>"
+    publish :function => :widgets, :type => "map (string, map (string, any))"
```
Comment (jreidinger) [inline src/modules/HttpServerWidgets.rb]: this is wrong. it should be `map (string, map <string, any>)` or does it really have as second argument method that takes string and any as parameters and return map?

**Comment by jreidinger** [comment]
and should it go also to SLE15 or master only?

File: src/modules/HttpServerWidgets.rb (hunk 1 above)
Comment (teclator) [reply src/modules/HttpServerWidgets.rb]: Uh, that is true

**Comment by teclator** [comment]
The original fix was master only.

**Pull request merged**